config/config.go

```diff
@@ -244,6 +252,16 @@ type LoadBalancer struct {
 	SecurityGroupIds []string `yaml:"securityGroupIds"`
 }
 
+type Plugins struct {
+	Rbac map[string]string `yaml:"rbac"`
```

Comment:
Can you make this explicit? This doesn't need free form params and helps with test coverage

EG
```golang
type Plugins struct {
  Rbac Rbac `yaml:"rbac"`
}

type Rbac struct {
  Enabled bool `yaml:"enabled"`
}
```